.azure/acceptance-pipeline.yaml

```diff
@@ -7,7 +7,16 @@ pr:
     include:
       - '*'
 
+parameters:
+  - name: releaseVersion
+    displayName: Release Version
+    type: string
+    # If releaseVersion == latest then the version is extracted from release.version file in root directory of the repo
```

Comment:
Is this still happening? It looks like the use of the `release.version` file is removed in this PR. (For the record, I do not think it is needed => it would be either latest or the parameter)

Reply:
ye, I just forgot to change the comment after some minor changes during the PR. Will fix it.